packages/app/src/cli/services/deploy/mode.ts

```diff
@@ -48,7 +48,7 @@ function displayDeployUnifiedBanner() {
 
 function displayDeployLegacyBanner(packageManager: PackageManager) {
   renderInfo({
-    headline: 'Deployments 2.0 available now.',
+    headline: 'Deployments 2.0 available now',
```

Comment:
isn't there an eslint rule forcing to add a `.` at the end of every headline?

Reply:
I guess not. Anyway there are differences about the use of the ` .` at the end of the line depending on using `legacy` or `unified` apps  [Figma](https://www.figma.com/file/dilbrKb0vthWTErdYgUunQ/Unified-App-Deployment?type=design&node-id=7469-45619&t=ckmT35CaPVOb8wmH-0)

I [mentioned](https://github.com/Shopify/cli/pull/2188#issuecomment-1594201890) this lack of consistency but we need to confirm if all the headers should have period or not @jessehoek @ani-shopify

Reply:
Ok, I will check that

Reply:
Just fetched the branch and it does complain on my machine about the broken rule. Maybe you need to restart ESLint 😬